pkg/functions/testdata/testFunctionLocalSettings/.func/local.yaml

```diff
@@ -0,0 +1 @@
+remote: true
```

Comment:
This is a very good test.  When you've implemented `.Write`, however, this test file can be removed, because then you'll be testing based off of _behavior_ rather than _implementation_! 🎉